src/cli/embedded_help.rb

```diff
@@ -16,7 +16,7 @@ def self.get_absolute_path(p)
       absolute_path = File.expand_path p2
 
       # strip Windows drive letters
-      if /[A-Z]\:/.match(absolute_path)
+      if /[A-Za-z]\:/.match(absolute_path)
```

Comment:
Fix is simple enough, problem was a needle in a haystack though. Great detective work!

Reply:
Side note: I doubt this happens in real life, but perhaps we should anchor the pattern to the start of the string, like this `/^[A-Za-z]\:/.match(absolute_path.strip)`

Reply:
make sense to add in a regex anchor. like you said, very unlikely to hit that but can't hurt.